types/mongodb/index.d.ts

```diff
@@ -962,8 +963,10 @@ export interface WriteOpResult {
 //http://mongodb.github.io/node-mongodb-native/2.1/api/Cursor.html#~resultCallback
 export type CursorResult = any | void | boolean;
 
+type Default = { [key: string]: any };
+
 //http://mongodb.github.io/node-mongodb-native/2.1/api/Cursor.html
-export class Cursor<T> extends Readable {
+export class Cursor<T = Default> extends Readable {
```

Comment:
You are right, I forgot to update the `AggregationCursor`. I'll try to do it today. Thanks for the reminder!